doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc

```diff
@@ -18,6 +18,19 @@ If you used a custom tag for the image, set the *Image tag* field to this tag.
 +
 . If you created the secret for Smart Router according to the instructions in <<secrets-smartrouter-create-proc_{context}>>, enter the name of the secret in the *Secret* field.
 . Optional: Enter the number of replicas for the Smart Router in the *Replicas* field.
+. Optional: To set a custom hostname for the external route, enter a domain in the *Custom hostname to be used on the Smart Router external Route* field, formatted as in the following example:
++
+----
+`smartrouter.example.com`
+----
++
+[NOTE]
+====
+The custom hostname must be valid and resolvable.
+
+To change the custom hostname, you can modify the `routeHostname` property.
```

Comment:
I can see that you are writing this sentence in the next line. Any specific reason for that? Can we skip the gap? This is for all the notes you have mentioned above. 
```suggestion
[NOTE]
====
The custom hostname must be valid and resolvable. To change the custom hostname, you can modify the `routeHostname` property.
====
```

Reply:
> I can see that you are writing this sentence in the next line. Any specific reason for that? Can we skip the gap? This is for all the notes you have mentioned above.

Reason is that their subject is slightly different, the first is general and about hostname validity while the second is about the property name in the config file. Thoughts?

Reply:
Thanks, Kalyani and Filippe!

Reply:
Going to keep them separate for now.